package.json

```diff
@@ -23,38 +28,44 @@
     "test:single": "./dev-env.sh jest --ci"
   },
   "dependencies": {
-    "@bugsnag/js": "7.6.0",
-    "@bugsnag/plugin-express": "7.6.0",
-    "@erento/nestjs-common": "4.0.0",
+    "@bugsnag/js": "7.16.1",
+    "@bugsnag/plugin-express": "7.16.1",
+    "@erento/eslint-plugin-erento-rules": "1.0.0",
+    "@erento/nestjs-common": "file:erento-nestjs-common-4.0.0.tgz",
     "@google-cloud/profiler": "4.1.3",
```

Comment:
there are newer deps, why not using the latest ones?

![Snímek obrazovky 2022-02-23 v 8 52 18](https://user-images.githubusercontent.com/156132/155279623-fc5f3149-8a3f-4c7d-a54b-0ef341e60f26.png)